src/org/labkey/remoteapi/query/SaveRowsCommand.java

```diff
@@ -185,6 +194,34 @@ public void addRow(Map<String, Object> row)
         _rows.add(row);
     }
 
+    public AuditBehavior getAuditBehavior()
+    {
+        return _auditBehavior;
+    }
+
+    /**
+     * Used to override the audit behavior for the schema/query
```

Comment:
I don't know if we want to explain here that this behavior will not be honored if there has already been customization of the audit behavior in the Java code or the XML.  We probably don't explain that in other APIs either.

Reply:
done